mobx_codegen/test/mobx_codegen_test.dart

```diff
@@ -34,6 +34,10 @@ void main() {
       await compareFiles('./data/valid_input.dart', './data/valid_output.dart');
     });
 
+    test('generates for a class mixing Store with annotation @StoreConfig(toString: true)', () async {
```

Comment:
StoreConfig is camel case: `storeConfig`. Its the annotation that end users will be using